src/Synthesiser.cpp

```diff
@@ -125,7 +125,7 @@ bool useNoIndex() {
     static bool flag = std::getenv(ENV_NO_INDEX);
     static bool first = true;
     if (first && flag) {
-        std::cout << "WARNING: indices are ignored!\n";
+        std::cout << "WARNING: indexes are ignored!\n";
```

Comment:
indices is the more common usage, outside of the US, and generally preferred in math and technical contexts. Both are acceptable, but why change?

Reply:
I would prefer US spelling - it is the dominant version of English.